README.md

```diff
@@ -21,3 +21,35 @@ In order to open traces you need a trace server running on the same machine as t
 2. From the repo root:  `yarn start:browser` or `yarn start:electron`
 3. Go to http://localhost:3000 or use the Electron application
 
+## Using the trace extension
+This section describes how to operate the Theia trace extension to view and analyze traces. The UI, view interactions, and UX are prelimiary and subject to revisions in the future.
+
+![theia-trace-extension](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-0.0.1.png)
+
+### Open trace(s)
+To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With->Open Trace(s)**.
+
+![Open trace(s)](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-open-traces-0.0.1.png)
+
+If the selection is a single file, then the tool will open the file directly as an experiment.
+
+If the selection is a directory, then the tool will look for traces in **Common Trace Format (CTF)** format, such as **Linux Tracing Toolkit traces (LTTng)** Kernel and UST (Userspace) traces, and open all found CTF traces as an experiment. With this you're able to open e.g. LTTng Kernel and UST Traces at the same time.
+
+Note that an experiment is a collection of traces that are analyzed together where each event of each trace in the experiment is analyzed in chronological order.
+
+The example Trace Compass trace server above supports LTTng Kernel and UST traces. Example LTTng traces can be retrieved from the [Trace Compass Tutorials](https://github.com/tuxology/tracevizlab). Just download the archive [TraceCompassTutorialTraces](https://github.com/tuxology/tracevizlab/blob/master/labs/TraceCompassTutorialTraces.tgz), extract them into a local directory on your computer.
+
+### Open the Trace Explorer
+To open the **Trace Explorer**, select menu **View** from the top-level menu and then select **Trace Explorer** in the list of views. Then the **Trace Explorer** icon will be added on the right side below the **File Explorer** Icon. 
+
+Select the **Trace Explorer** icon to switch to the trace explorer.
+
+![Trace Explorer](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-trace-explorer-0.0.1.png)
+
+Now, you will see 3 sections: **Opened eperiments**, **Available analysis**” and **Time Graph Tooltip**. Select the experiment you opened and the available analysis will be populated. Now you can select different analysis and the graphs will be added into a single container view for that experiment.
```

Comment:
Opened ex*periments, additional " after available analysis

Reply:
Will update this typos